docs/specs/clients/sign/client-api.md

```diff
@@ -74,6 +74,24 @@ abstract class Client {
     reason: Reason;
   }): Promise<void>;
 
+  // ---------- Wallet Authentication Methods ----------------------------------------------- //
+
+  // for proposer to create an authenticated pairing
```

Comment:
authenticated pairing?
we have never covered that concept 😅

Reply:
fixed the wording